index.js

```diff
@@ -50,7 +51,7 @@ export default class ClapprStats extends ContainerPlugin {
     this.listenToOnce(this.container, Events.CONTAINER_PLAY, this.onFirstPlaying)
     this.listenTo(this.container, Events.CONTAINER_PLAY, this.onPlay)
     this.listenTo(this.container, Events.CONTAINER_PAUSE, this.onPause)
-    this.listenTo(this.container, Events.CONTAINER_TIMEUPDATE, (e) => e.current > 0 && this.onContainerUpdateWhilePlaying())
+    this.listenTo(this.container, Events.CONTAINER_TIMEUPDATE, (e) => this.container.playback.isPlaying() && this.onContainerUpdateWhilePlaying())
```

Comment:
Oh, and this was also necessary when using hls.js playback. From times to times, `e.current` brings `0`. Specially testing with a poor internet connection.